frontend/src/features/admin-form/create/end-page/EndPageContent.tsx

```diff
@@ -8,10 +8,11 @@ import {
   FormResponseMode,
 } from '~shared/types'
 
+import { EndPageBlock } from '~components/FormEndPage/EndPageBlock'
+
 import { useAdminForm } from '~features/admin-form/common/queries'
 import { PREVIEW_MOCK_UINFIN } from '~features/admin-form/preview/constants'
 import { useEnv } from '~features/env/queries'
-import { EndPageBlock } from '~features/public-form/components/FormEndPage/components/EndPageBlock'
 import { PaymentsThankYouSvgr } from '~features/public-form/components/FormEndPage/components/PaymentsThankYouSvgr'
```

Comment:
these two seem to also be likely candidates to move over to the generic folders together?

Reply:
yup that makes sense! I'll add these 2 Svgr files to the generic components folder `FormEndPage` too